docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -23,7 +23,14 @@ title: How to use go-plugin-runner in APISIX Ingress
 
 ## Proposal Description
 
-Based on version 0.3 of the go-plugin-runner plugin and version 1.4.0 of APISIX Ingress, this article goes through building the cluster, building the image, customizing the helm chart package, and finally, deploying the resources. It is guaranteed that the final result can be derived in full based on this documentation.
+Based on version 0.3 of the go-plugin-runner plugin and version 1.4.0 of APISIX Ingress, this article goes through steps as follows:
+
+1. Build the cluster;
+2. Build the image;
```

Comment:
Build a container image that includes plugin-runner

Reply:
Solved~